### PR TITLE
BWDB-14422: Make campaign Vertical as not required

### PR DESCRIPTION
**Pull request by nirmitpatel**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

File: site/docs/messaging/campaign-management/csp/campaign-api.mdx [hunk 1]
```diff
@@ -87,7 +87,7 @@ HTTP/1.1 429 Too Many Requests
 | Campaign                   | Mandatory | Description                                                    |
 |:---------------------------|:----------|:---------------------------------------------------------------|
 | `BrandId`                  | Yes       |  Alphanumeric identifier of the brand associated with this campaign  |
-| `Vertical`                 | Yes       | 	Business/industry segment of this campaign. Must be of defined valid types |
+| `Vertical`                 | No        | 	Business/industry segment of this campaign. Must be of defined valid types or left blank/set to null |
```
Comment (mrmeyers99) [inline site/docs/messaging/campaign-management/csp/campaign-api.mdx:90]: shouldn't this be in the api docs?

Comment (nirmitpatel) [reply site/docs/messaging/campaign-management/csp/campaign-api.mdx:90]: Campaigns don't have API docs when it comes to CSP. It's only this guide

**Review by adarkbandwidth — DISMISSED** [review]
Just update wording to reflect the lack of any validation: "Optional business/industry segment of this campaign (any value allowed)"

**Review by AswiniKumarV — APPROVED** [review]

**Review by adarkbandwidth — APPROVED** [review]

**Review by ajrice6713 — APPROVED** [review]

File: site/docs/messaging/campaign-management/csp/campaign-api.mdx (hunk 1 above)
Comment (mrmeyers99) [reply site/docs/messaging/campaign-management/csp/campaign-api.mdx]: We need to add these endpoints to the API docs.  Can you create a Jira for it?

Comment (nirmitpatel) [reply site/docs/messaging/campaign-management/csp/campaign-api.mdx]: Created https://bandwidth-jira.atlassian.net/browse/MV-7841

**Review by mrmeyers99 — APPROVED** [review]

**Pull request merged**